client/ayon_nuke/api/plugin.py

```diff
@@ -222,10 +224,7 @@ def update_instances(self, update_list):
 
             # update instance node name if product name changed
             if "productName" in changes.changed_keys:
-                instance_node["name"].setValue(
-                    changes["productName"].new_value
-                )
-
+                update_write_node_filepath(created_inst, changes)
```

Comment:
This feels like an odd change?

So this should ALSO change the filepath? If so, should it also do so on task change, etc? If so, we don't need this if statement here. and we can just always call it - and then also do not need to implement `update_instances` on the other plugin?